R/io.R

```diff
@@ -103,6 +110,32 @@ read_civis.sql <- function(x, database = NULL, using = utils::read.csv,
   })
 }
 
+#' @describeIn read_civis Return run outputs of a \code{civis_script} as a named list.
+#' @param regex Regex of matching run output names.
+#' @details
+#' If \code{using = NULL}, \code{read_civis.civis_script}
+#' will return all JSONValues with name matching \code{regex}.
+#' Otherwise all File run outputs matching \code{regex} will be read into memory
+#' with \code{using}.
+#' Results are always a named list. If the script has no outputs, an error will be signalled.
+#' @export
+read_civis.civis_script <- function(x, regex = NULL, using = NULL, ...) {
+  output <- fetch_output(x, regex = regex)
+  if (is.null(using)) {
```

Comment:
What happens if a user calls this on a script with a file_id output but doesn't specify `using`?